nsxt/resource_nsxt_manager_cluster.go

```diff
@@ -314,6 +258,34 @@ func getHostCredential(m interface{}) (string, string) {
 	return username, password
 }
 
+func getMatchingIPVersion(ip string, hostIPs []string) string {
+	needIPv4 := (net.ParseIP(ip)).To4() != nil
+
+	for _, hostIP := range hostIPs {
+		isIPv4 := (net.ParseIP(hostIP)).To4() != nil
```

Comment:
are 100% sure hostIp is always a valid Ip address? otherwise we might first need to check ParseIP result.

Reply:
We validate hostIP in `resolveHostIPs`, which is happening before this call